constraints.tex

```diff
@@ -0,0 +1,75 @@
+\section{Constraints and Limitations}
+
+\begin{note}
+  Wil: Describes the limitations and the constraints which apply to CU level tests of the system. lack of computing resources may mean that datasets are smaller or that full accuracy cannot be achieved. Explain what must be validated in the DPC tests.
+\end{note}
+
+\begin{itemize}
+
+\item Verification is being done on the basis of precursor data sets such as HSC, and eventually with engineering data from the LSST arrays. These are just a proxy for full-focal-plane on-site LSST data.
+
+\item Metric measurements and operational rehearsals during construction may not involve critical operational systems that are still in development. For example, while computational performance is being measured, computationally dominant algorithmic steps such as deblending and multi-fit are only modeled, since they have not yet been implemented; operational rehearsals are done without the factory LSST workflow system; etc.
+
+\end{itemize}
+
+\subsection{Requirements Traceability Constraints}
+
+\begin{note}
+  I felt a summary of the current state of play being verified could be useful to Wil. We don't have to leave it in the final document --FE
+\end{note}
+
+\subsubsection{Scientific}
+
+Some science requirements are captured in \citeds{LSE-29} (aka \LSR) and  flow down to \citeds{LSE-30} (aka \OSS) ; some also exist in \citeds{LSE-163} (aka \DPDD) and will flow down in \citeds{LSE-61} (aka \DMSR).
+
+\begin{note}
+Flowdown is not complete, TJ is working on this.
+\end{note}
+
+\subsubsection{Computational}
+
+There are requirements in \citeds{LSE-61} (aka \DMSR) which captures the \citeds{LSE-30} (\OSS) requirements that DM is responsible for. \textit{In practice \citeds{LSE-63} (the QA document) has not been flown down to \citeds{LSE-61}}. These are:
+
+\begin{itemize}
+
+\item The primary computational performance flown down from \citeds{LSE-29} (\LSR) is OTT1 which is the requirement to issue an alert within 60 seconds of exposure end.\dmreq{0004}\lsrreq{0101}
+
+\item Another requirement flows down from \citeds{LSE-29} is calculation of orbits within 24 hours of the end of the observing night\dmreq{0004}\lsrreq{0104}\reqparam{L1PublicT}
+
+\item There is a new (not yet baselined?) requirement for the calibration pipeline to reduce calibration observations within 1200 seconds
+
+\item A nightly report on data quality, data management system performance and a calibration report have to be generated with 4 hours of the end of the night\dmreq{0096}\reqparam{dq\-Report\-Compl\-Time}
+
+\end{itemize}
+
+Note that there are no computational requirements on individual technical components eg. data processing cluster availability, database data retrieval speeds, etc. There is an upper limit on acceptable data loss, and there is a network availability requirement.
```

Comment:
Some of the performance metrics are used as assumptions in the baselined sizing model LSE-81/LSE-82 but need to be materialized in DMSR (or component requirements documents).